tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/DocumentProvision.cs

```diff
@@ -345,86 +353,149 @@ public GenericResponseVM PerformContentCheck(string clientUrl, string folderUrl,
             var searchResultsVM = await docRepository.GetDocumentsAsync(searchRequestVM);
 
             if (searchResultsVM.TotalRows > 0)
-            {
-                IList<DocumentData> documentDataList = new List<DocumentData>();
+            {                
+                dynamic documentDataList = new List<dynamic>();

```

Comment:
Instead of concrete DocumentData object, dynamic object of type Expando object has been created to cater the needs of both matter center and project center